vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -86,13 +86,14 @@ func (calc *UpdatePriorityCalculator) AddPod(pod *apiv1.Pod, recommendation *vpa
 	updatePriority := calc.getUpdatePriority(pod, processedRecommendation)
 
 	quickOOM := false
-	if len(pod.Status.ContainerStatuses) == 1 {
-		terminationState := pod.Status.ContainerStatuses[0].LastTerminationState
+	for _, cs := range pod.Status.ContainerStatuses {
```

Comment:
There is a possibility to switch off autoscaling for a given container, we probably don't want to react to ooms in those.